docs/docs/policies.mdx

```diff
@@ -85,46 +85,65 @@ The Transformer Embedding Dialogue (TED) Policy is described in
 This policy has a pre-defined architecture, which comprises the
 following steps:
 
-* concatenate user input (user intent and entities), previous system actions, slots and active forms for each time
+1. concatenate user input (user intent and entities), previous system actions, slots and active forms for each time
   step into an input vector to pre-transformer embedding layer;
 
-* feed it to transformer;
+2. feed it to transformer;
 
-* apply a dense layer to the output of the transformer to get embeddings of a dialogue for each time step;
+3. apply a dense layer to the output of the transformer to get embeddings of a dialogue for each time step;
 
-* apply a dense layer to create embeddings for system actions for each time step;
+4. apply a dense layer to create embeddings for system actions for each time step;
 
-* calculate the similarity between the dialogue embedding and embedded system actions.
-  This step is based on the [StarSpace](https://arxiv.org/abs/1709.03856) idea.
+5. calculate the similarity between the dialogue embedding and embedded system actions.
+   This step is based on the [StarSpace](https://arxiv.org/abs/1709.03856) idea.
 
-It is recommended to use `state_featurizer=LabelTokenizerSingleStateFeaturizer(...)`
-(see [Featurization of Conversations](./policies.mdx#featurization-conversations) for details).
+It is recommended to use `LabelTokenizerSingleStateFeaturizer`
+(see [Featurization of Conversations](./policies.mdx#featurizers) for details).
+
+```yaml-rasa title="config.yml"
+policies:
+  - name: "TEDPolicy"
+    featurizer:
+    - name: MaxHistoryTrackerFeaturizer
+      max_history: 5
+      state_featurizer:
+        - name: LabelTokenizerSingleStateFeaturizer
+```
 
 **Configuration:**
 
-Configuration parameters can be passed as parameters to the `TEDPolicy` within the configuration file.
-If you want to adapt your model, start by modifying the following parameters:
+You can pass configuration parameters to the `TEDPolicy` using the `config.yml` file.
+If you want to fine-tune your model, start by modifying the following parameters:
 
 * `epochs`:
   This parameter sets the number of times the algorithm will see the training data (default: `1`).
   One `epoch` is equals to one forward pass and one backward pass of all the training examples.
   Sometimes the model needs more epochs to properly learn.
   Sometimes more epochs don't influence the performance.
-  The lower the number of epochs the faster the model is trained.
+  The lower the number of epochs the faster the model is trained. Here is how the config would look like:
 
-
-:::note configure epochs
-Pass an appropriate number, for example 50,  of `epochs` to the `TEDPolicy`, otherwise the policy will
-be trained only for `1` epoch.
-
-:::
+  ```yaml-rasa title="config.yml"
+  policies:
+  - name: TEDPolicy
+    # if not an appropriate number, it'll use the default value
```

Comment:
I don't think this comment is necessary, its not checking for appropriateness (except maybe negative) -- the old comment was basically "pass a number or itll use the default" but also make sure your number is appropriate